trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, a TC represents the interests of both
+their InnerSource community and the products the community is building. So as
+a TC, you'll have both tech oriented and community oriented responsibilities
+and we'll explore both of these dimensions in the following sections.
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
+    - makes contributions
+    - sometimes part of the community
+    - sometimes send by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as Guests. 
+    - responsible for "fitting in"/conforming to the host team and it's 
+      expectations and processes
+    - similar to an Apache Developer or committer
+
+- Trusted Committer
+    - Sometimes referred to as a host.
+    - In this analogy, he is both resonsible for building the house and setting
+      the house rules to make sure his guests are comfortable.
+    - In comparison to the Contributor, a TC has rights to push code closer to
+      production
+- Product Owner
+    - Responsible for defining and prioritizing requirements and stories for
+      the community to implement.
+    - Also interacts directly and frequently with the trusted committers
+    - Distinction between the guest and host PO - have to negotiate
+      requirements, if necessary.
+    - Depending on the type of InnerSource community, there might not be a PO
+      from the outset. Especially true in case of innovation oriented and
+      bottom-up driven communities. POs come into play when a productization
+      of an InnerSource communities products becomes relevant.
+
+    ? In the Guest/Host analogy, what would be the analogon of a PO?
+
+## Why role names matter
```

Comment:
I think this could be a better way to introduce the trusted committer learning path.

You could re-organize start with the general bits below into two categories:
1. why roles matter, 
1. Why trusted committers matter 

This would segway nicely into your main topic above.

Reply:
@mrsanz : I actually started with this reversed order (the [current playbook](https://docs.google.com/spreadsheets/d/1TJnrCIudz-R2EA_sresxSxXe5wRvGZKHu2M5Jy_22uM/edit#gid=0) still has it in this order). While writing it, though, it didn't feel right to do it this way. I'd like to start by introducing the role and _after_ I've done that, elaborate on why the naming is important. Feels more natural to me. What does everyone else think?

Reply:
You make a good point, @mrsanz . What does everyone else think?

Reply:
I like the order in the current document.  I think that we can assume that people are interacting with this training because they _want_ to hear what we have to say.  Given that level of reception and desire, no need to assume a defensive posture out of the gate.